src/Params/ADnoteParameters.cpp

```diff
@@ -1024,7 +1086,11 @@ void ADnoteGlobalParam::getfromXML(XMLwrapper& xml)
         FreqLfo->getfromXML(xml);
         xml.exitbranch();
 
+        xml.enterbranch("FREQUENCY_SEQ");
```

Comment:
Load failure happens here. you're unconditionally trying to enter a branch which may or may not exist in the source savefile.